### PR TITLE
Creates Government SDK Intro Guide

### PR DESCRIPTION
**Pull request by deeleeramone**
added a new file, `government.md`

### CODE REVIEW

File: website/content/sdk/guides/intros/stocks/government.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+```
+
+The examples below will assume that the block above is included at the top of the file.
+
+### Toplobbying
+
+This command shows who is spending money in Washington and what their intentions are.
+
+```python
+gov.toplobbying()
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/government.md:48]: Convert `gov` to `openbb.stocks.gov`

File: website/content/sdk/guides/intros/stocks/government.md [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+| 497 | 2022-10-20 | HGV      | HILTON GRAND VACATIONS                            |    20000 | Financial Institutions/Investments/Securities | GENERAL DISCUSSIONS RELATED TO DEBT SERVICING                               |
+| 498 | 2022-10-20 | SO       | SOUTHERN CALIFORNIA TRIBAL CHAIRMEN'S ASSOCIATION |    40000 | Indian/Native American Affairs                | Fee to trust issues. Issues surrounding federal programs for Indian tribes. |
+| 499 | 2022-10-20 | ONCS     | ONCOSEC MEDICAL INCORPORATED                      |        0 |                                         |                                                                             |
+
+### Contracts
+
+Get a list of contracts awarded to a company and the agency authorizing it.
+
+```python
+gov.contracts('AMT')
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/government.md:75]: Convert `gov` to `openbb.stocks.gov`

File: website/content/sdk/guides/intros/stocks/government.md [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+
+### Toplobbying
+
+This command shows who is spending money in Washington and what their intentions are.
+
+```python
+gov.toplobbying()
+```
+
+|    | Date       | Ticker   | Client                          |   Amount | Issue                | Specific_Issue  |
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/government.md:51]: Please shorten this table, it is currently very large. I'd suggest doing `limit=3` for example. People get the gist of it with a few inputs.

File: website/content/sdk/guides/intros/stocks/government.md [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+| 13 | 2022-10-01 | TUCSON MOUNTAIN                                                                                                                                                                                 | DEPARTMENT OF THE TREASURY (TREAS)    | 10056.8  |
+| 14 | 2022-09-10 | TOWER SITE RENTAL  REQUEST IS SUBJECT TO THE AVAILABILITY OF FY2023 FUNDS                                                                                                                       | DEPARTMENT OF JUSTICE (DOJ)           |     0    |
+| 15 | 2022-05-21 | FISCAL YEAR 2021 ANTENNA SITE(S) #043, LEASE RENEWAL - SUBJECT TO AVAILABILITY OF FUNDS.                                                                                                        | DEPARTMENT OF HOMELAND SECURITY (DHS) |     0    |
+
+### Histcont
+
+Get the historic (from 2008) quarterly sum paid to a company by the US Treasury department.
+
+```python
+gov.histcont('AMT')
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/government.md:102]: Convert `gov` to `openbb.stocks.gov`

File: website/content/sdk/guides/intros/stocks/government.md [hunk 5]
```diff
@@ -0,0 +1,167 @@
...
+
+### Contracts
+
+Get a list of contracts awarded to a company and the agency authorizing it.
+
+```python
+gov.contracts('AMT')
+```
+
+|    | Date       | Description                                                                                                                                                                                     | Agency                                |   Amount |
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/government.md:78]: Please reduce the size of this table, e.g. with `limit=5`. It is currently too large.

File: website/content/sdk/guides/intros/stocks/government.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+| 53 |     3 |   2008 |   240877 |
+| 54 |     2 |   2008 |   907099 |
+| 55 |     1 |   2008 |   320620 |
+
+### Lobbying
+
+Find the records of a company's lobbying efforts.
+
+```python
+gov.lobbying('AMT', limit = 500)
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/government.md:120]: Convert `gov` to `openbb.stocks.gov`

File: website/content/sdk/guides/intros/stocks/government.md [hunk 7]
```diff
@@ -0,0 +1,167 @@
...
+```
+
+|    | Date       | Ticker   | Client                     |   Amount | Issue                                  | Specific_Issue                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/government.md:123]: Please reduce the size of this table to e.g. `limit=3`.

File: website/content/sdk/guides/intros/stocks/government.md [hunk 8]
```diff
@@ -0,0 +1,167 @@
...
+| 172 | 2009-02-12 | AMT      | AMERICAN TOWER CORPORATION |        0 | Telecommunications | Broadband access incentive provisions in H.R. 1, The American Recovery and Reinvestment Act of 2009. |
+| 173 | 2004-02-20 | AMT      | AMERICAN TOWER CORP        |        0 | Telecommunications |                                                                                                      |
+| 174 | 2004-02-20 | AMT      | AMERICAN TOWER CORP        |    20000 | Telecommunications |                                                                                                      |
+
+### Lastcontracts
+
+Find out who is selling ink, toner cartridges, or other seemingly mundane items, to the Federal government.
+
+```python
+gov.lastcontracts()
```
Comment (JerBouma) [inline website/content/sdk/guides/intros/stocks/government.md:153]: Convert `gov` to `openbb.stocks.gov`

File: website/content/sdk/guides/intros/stocks/government.md [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+
+## Examples
+
+Let's import the SDK in a way that reduces the amount of typing required to access the module.
+
+### Import Statements
+
+```python
+from openbb_terminal.sdk import openbb
+gov = openbb.stocks.gov
```
[the remark this answers is not in the document]
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/government.md:38]: Done

File: website/content/sdk/guides/intros/stocks/government.md (hunk 2 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/government.md:75]: Done

File: website/content/sdk/guides/intros/stocks/government.md (hunk 4 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/government.md:102]: Done

File: website/content/sdk/guides/intros/stocks/government.md (hunk 6 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/government.md:120]: done

File: website/content/sdk/guides/intros/stocks/government.md (hunk 8 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/government.md:153]: done

File: website/content/sdk/guides/intros/stocks/government.md (hunk 3 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/government.md:51]: done

File: website/content/sdk/guides/intros/stocks/government.md (hunk 5 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/government.md:78]: done

File: website/content/sdk/guides/intros/stocks/government.md (hunk 1 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/government.md:48]: done

File: website/content/sdk/guides/intros/stocks/government.md (hunk 7 above)
Comment (deeleeramone) [reply website/content/sdk/guides/intros/stocks/government.md:123]: done

**Review by JerBouma — APPROVED** [review]

**Pull request merged**